azure-pipelines.yml

```diff
@@ -566,33 +555,6 @@ jobs:
     - bash: ./scripts/ci/test_extensions.sh
       displayName: 'Load extensions'
 
-- job: RunAutomation20180301Python3_8
-  displayName: Run Automation Python 3.8, Profile 2018-03-01
-  dependsOn: BuildPythonWheel
-
-  pool:
-    vmImage: 'ubuntu-16.04'
-  steps:
-  - task: UsePythonVersion@0
-    displayName: 'Use Python 3.8'
-    inputs:
-      versionSpec: 3.8
-  - task: DownloadPipelineArtifact@1
-    displayName: 'Download Python Wheels'
-    inputs:
-      TargetPath: '$(Build.ArtifactStagingDirectory)/pypi'
-      artifactName: pypi
-  - bash: |
-      pip install --force-reinstall wheel
-  - task: Bash@3
-    displayName: 'Run Test Automation'
-    inputs:
-      targetType: 'filePath'
-      filePath: ./scripts/ci/test_automation.sh
-    env:
-      AZURE_CLI_TEST_TARGET_PROFILE: '2018-03-01'
-      ADO_PULL_REQUEST_TARGET_BRANCH: $(System.PullRequest.TargetBranch)
```

Comment:
why cannot we reuse these two environment?

Reply:
They have became parameters while instantiating the template `.azure-pipelines/template/automation_test.yml`.